test/grant-manager-token-timeout-spec.js

```diff
@@ -2,54 +2,86 @@
 
 const GrantManager = require('../middleware/auth-utils/grant-manager');
 const Config = require('../middleware/auth-utils/config');
-const test = require('tape');
+const t = require('tap');
```

Comment:
IHMO, test is better

Reply:
If you want this then have a look at the attached file as it shows what the code would look like and it's not nice IMHO as it would require all of the references in the code to 't' to become 'test' and as such would mean allot more changes and make the code potentially harder
[test.js.txt](https://github.com/keycloak/keycloak-nodejs-connect/files/6493133/test.js.txt)
 to understand as 't' and 'test' both become 'test'